### PR TITLE
Add service maintenance page

### PR DESCRIPTION
**Pull request by RMcVelia**
### Context

Add service maintenance page
https://trello.com/c/E7mn31J2/2003-register-add-maintenance-page

### Changes proposed in this pull request

Add maintenance page scripts from template
Add enable/disable maintenance workflows

### Guidance to review

tested in qa env

image
 
### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [ ] Do we need to send any updates to DQT as part of the work in this PR?
- [ ] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by RMcVelia** [comment]
[Register - add maintenance page](https://trello.com/c/E7mn31J2/2003-register-add-maintenance-page)

**Review by defong — APPROVED** [review]
Amazing, can we have a demo, pretty please 👏👏👏

This looks like something that can be extracted to it lonesome repo and be made shareable.

**Comment by RMcVelia** [comment]
> Amazing, can we have a demo, pretty please 👏👏👏
> 
> This looks like something that can be extracted to it lonesome repo and be made shareable.

It's built from the template in the teacher-services-cloud repo, see https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/maintenance-page.md#initial-configuration

**Review by tomtrentham — APPROVED** [review]
This is great! 🚀 
Just one suggestion on the page footer.

File: maintenance_page/html/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        </div>
+      </main>
+    </div>
+    <footer role="contentinfo" class="govuk-footer govuk-!-display-none-print">
+      <div class="govuk-width-container">
+        <div class="govuk-footer__meta">
+          <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+            <h2 class="govuk-heading-m">Get help</h2>
+            <p class="govuk-body-s govuk-!-margin-bottom-1">Call 0800 389 2500 or <a class="govuk-link govuk-footer__link" href="https://getintoteaching.education.gov.uk/help-and-support?utm_source=apply-for-teacher-training.service.gov.uk&amp;utm_medium=referral&amp;utm_campaign=candidate_interface%2Fstart_page-create_account_or_sign_in&amp;utm_content=candidate_not_logged_in">chat online</a>.</p>
+            <p class="govuk-body-s govuk-!-margin-bottom-1">Monday to Friday, 8:30am to 5:30pm (except public&nbsp;holidays)</p>
```
[the remark this answers is not in the document]
Comment (RMcVelia) [reply maintenance_page/html/index.html]: thanks, updated as suggested

**Pull request merged**